
If we run depmod than with version argument - however the linux package
does it at the end - so we skip it at all ,-)

--- ipw2100-1.0.5/Makefile.orig	2005-03-25 15:24:40.809653152 +0100
+++ ipw2100-1.0.5/Makefile	2005-03-25 15:25:58.928777240 +0100
@@ -205,14 +205,14 @@
 install: modules
 	install -d $(KMISC)
 	install -m 644 -c $(addsuffix .ko,$(list-m)) $(KMISC)
-	/sbin/depmod -a
+	@echo "T2: not running depmod in build."
 	@echo "Don't forget to copy firmware to your hotplug's firmware direectory and have the "
 	@echo "hotplug tools in place."
 	@echo "See INSTALL for more information."
 
 uninstall:
 	rm -rf $(addprefix $(KMISC),$(addsuffix .ko,$(list-m)))
-	/sbin/depmod -a
+	@echo "T2: not running depmod in build."
 
 endif # End of internal build
 
